modules/react/_examples/stories/examples/GlobalHeader.tsx

```diff
@@ -1,41 +1,161 @@
 import * as React from 'react';
-import {styled, createComponent, dubLogoBlue} from '@workday/canvas-kit-react/common';
+import {
+  AccessibleHide,
+  AriaLiveRegion,
+  composeHooks,
+  createComponent,
+  createElemPropsHook,
+  createSubcomponent,
+  ExtractProps,
+  useUniqueId,
+} from '@workday/canvas-kit-react/common';
 import {colors, depth, space, type} from '@workday/canvas-kit-react/tokens';
-
+import {system} from '@workday/canvas-tokens-web';
+import {createStyles, cssVar, px2rem} from '@workday/canvas-kit-styling';
 import {
   notificationsIcon,
   inboxIcon,
   justifyIcon,
   assistantIcon,
+  searchIcon,
 } from '@workday/canvas-system-icons-web';
 
-import {TertiaryButton, Hyperlink} from '@workday/canvas-kit-react/button';
+import {SecondaryButton, TertiaryButton} from '@workday/canvas-kit-react/button';
 import {Avatar} from '@workday/canvas-kit-react/avatar';
 import {Flex, FlexProps} from '@workday/canvas-kit-react/layout';
-import {SearchForm} from '@workday/canvas-kit-labs-react/search-form';
+import {Tooltip} from '@workday/canvas-kit-react/tooltip';
+import {Combobox, useComboboxModel, useComboboxInput} from '@workday/canvas-kit-react/combobox';
+import {InputGroup, TextInput} from '@workday/canvas-kit-react/text-input';
+import {StyledMenuItem} from '@workday/canvas-kit-react/menu';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {CountBadge} from '@workday/canvas-kit-react/badge';
 
 interface HeaderItemProps extends FlexProps {}
+interface LiveCountBadgeProps extends FlexProps {
+  cnt: number;
+}
+
+const tasks = ['Request Time Off', 'Create Expense Report', 'Change Benefits'];
+
+function negate(value: string, fallback?: string) {
+  return `calc(${cssVar(value, fallback)} * -1)`;
+}
+
+const styleOverrides = {
+  headerWrapper: createStyles({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    boxSizing: 'border-box',
+    ...type.levels.subtext.large,
+    WebkitFontSmoothing: 'antialiased',
+    MozOsxFontSmoothing: 'grayscale',
+    backgroundColor: colors.frenchVanilla100,
```

Comment:
Let's update this values to use your new tokens `system.color.bg.default`. Here's a renferece: https://workday.github.io/canvas-tokens/?path=/story/visual-tests-system-tokens-colors--background